sky/clouds/cudo.py

```
@@ -21,6 +21,27 @@
 ]
 
 
+def _load_cudo_api_module() -> Tuple[typing.Any, Type[Exception]]:
+    """Returns the Cudo API module and its ApiException."""
+    errors = []
+    try:
+        from cudo_compute import cudo_api as module
+        from cudo_compute.rest import ApiException  # type: ignore
+        return module, ApiException
```

Ruff (0.14.2)

Consider moving this statement to an `else` block

(TRY300)